nemo/Cargo.toml

```diff
@@ -30,6 +30,7 @@ thiserror = "1.0"
 flate2 = "1"
 sanitise-file-name = "1.0.0"
 nom_locate = { version = "4.1.0", features = [ "runtime-dispatch-simd" ] }
+bytecount = "=0.6.4"
```

Comment:
ah, I would have suggested to do a patch version instead

Reply:
This is really just a temporary workaround, I don't want to keep this around once a fixed version is available.

Reply:
nvm, patches must point to a different location